administrator/components/com_guidedtours/src/Model/TourModel.php

```diff
@@ -355,6 +355,7 @@ public function duplicate(&$pks)
                                 'checked_out_time',
                                 'checked_out',
                                 'language',
+	                            'params',
```

Comment:
```suggestion
                                'params',
```
There was still one tab among the spaces for indentation. I suggest to use an editor or IDE which showsn you tabs and spaces in code.